docs/user-profile-and-api-key.md

```diff
@@ -0,0 +1,30 @@
+---
+id: user-profile-and-api-key
+title: User Profile & API Key
+---
+The ARGO User Profile page displays important user information including:
+- which email you have logged into the ARGO platform with
+- what your ARGO Member permission are
+- API Key management
+
+To get to your User Profile, click the initial icon in the top right corner and select Profile and Token.  
+## API Key
+Your authentication API Key uniquely identifies you, and your permissions, to all ARGO Data Platform applications, for example when using the Song or Score clients for data submission.
+
+You can get your authentication API Key from the [User Profile](https://platform.icgc-argo.org/user) > API Key section.  
+![Profile API Key](../img/data-access-user-profile-api-key.png)
+Some important things to note about your API Key:
+- ARGO API keys have a duration **30 days**.
+- This is a _personal key_.  **Never** share your user credentials with anyone else.
+- If you suspect that your API-key security has been compromised, simple regenerate it from this page.  All former API Keys will be revoked when you regenerate a token.
+
+## Member Permissions
+Member permissions are assigned based the role that has been granted to you in a specific program, the type of program you are a member of, or your application for access to controlled data.
```

Comment:
Member permissions are assigned based on your program role, the type of program you are a member of, and your access level for controlled data.

or should be and right?

Reply:
i dont think it actually matters, but will change to and.